Description: Fix minor spelling error in source code
Author: Marcos Fouces <marcos.fouces@gmail.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/


--- a/remote.c
+++ b/remote.c
@@ -652,7 +652,7 @@
 	  if (remote_command_count > 0)
 	    {
 	      fprintf (stderr,
-		"%s: the `-id' option must preceed all `-remote' options.\n",
+		"%s: the `-id' option must precede all `-remote' options.\n",
 		       progname);
 	      usage ();
 	      exit (-1);
